changes/noissue.12.fix.rst

```diff
@@ -0,0 +1,4 @@
+Development: Fix issue with minimum-constraints-develop.txt that was causing
+failure of make check_reqs because the package Levenshtein and
+python-Levenshtein are required by safety and are not in that file.  Added the
+minimum version constraint required by safety.
```

Comment:
The "python-Levenshtein" package has been added by safety 3.3.0 and will be removed again in safety 3.3.1 because it is GPL-licensed. See https://github.com/pyupio/safety/issues/682. Since "safety" is the only package using "python-levenshtein" (or its renamed package "levenshtein"), I suggest fixing this issue by excluding safety 3.3.0 instead of adding any of the "levenshtein" packages.

Reply:
Done